packages/ui-kit/src/components/SimpleFilter/SimpleFilter.types.ts

```diff
@@ -0,0 +1,32 @@
+import { DataPoolInput } from '../../helpers'
+import { AutocompleteProps } from '../Autocomplete/Autocomplete.types'
+import { ErrorFallbackProps } from '../ErrorFallback'
+import { DataComponentProps, QueryProps } from '../shared.types'
+
+export interface SimpleFilterQueryProps extends Omit<QueryProps, 'metric' | 'filters'> {
+  /** The column to fetch the unique values from */
+  columnName?: string
+
+  /** The Data Pool to be queried. A Data Pool ID or unique name can be provided */
+  dataPool?: DataPoolInput
+
+  /** The maximum number of values to return. It can be a number between 1 and 1,000. If the parameter is omitted, default value 10 is used */
+  maxValues?: number
+}
+
+export interface SimpleFilterProps extends ErrorFallbackProps, Omit<DataComponentProps, 'card'> {
+  /** Props that the autocomplete input will receive */
+  autocompleteProps?: Omit<AutocompleteProps, 'options'>
```

Comment:
Should it just be `autocomplete`? Is "Props" in `autocompleteProps` redundant?

Do we have the "Props" suffix in other prop names? If so, we can keep it consistent.

Reply:
Yes, we are using this suffix in other places so I think we can keep consistency.

Here's an example: https://github.com/propeldata/ui-kit/blob/main/packages/ui-kit/src/components/Leaderboard/Leaderboard.types.ts#L72-L75